tests/system/marathon_common_tests.py

```diff
@@ -39,7 +39,7 @@ def test_launch_docker_container():
 
     client = marathon.create_client()
     client.add_app(app_def)
-    shakedown.deployment_wait()
+    shakedown.deployment_wait(app_id=app_def["id"])
```

Comment:
for almost all this tests `app_def["id"]` is used at least 2x times in the test. It probably makes sense to extract it to a var e.g. `app_id = app_def["id"]`